tap_pendo/discover.py

```diff
@@ -63,16 +63,17 @@ def get_schema_property_type(schema_type):
     elif schema_type == '':
         return {}
 
-    raise Exception("No case matching JSON schema for property type: {}".format(schema_type))
+    raise Exception(
```

Comment:
Why it is showing changes in this file? If nothing changed then do not commit it.